src/main/java/net/imagej/legacy/plugin/MacroAutoCompletionProvider.java

```diff
@@ -217,7 +223,24 @@ public String getToolTipText(final RTextArea textArea, final MouseEvent e) {
 	}
 
 	protected boolean isValidChar(char ch) {
-		return Character.isLetterOrDigit(ch) || ch == '_' || ch == '.';
+		return Character.isLetterOrDigit(ch) || ch == '_' || ch == '.' || ch == '"' || ch == '(';
 	}
 
+	void addModuleCompletions(ModuleService moduleService) {
+		if (this.moduleService == moduleService) {
+			return;
+		}
+		this.moduleService = moduleService;
+
+		for (ModuleInfo info : moduleService.getModules()) {
+			if(info.getMenuPath().getLeaf() != null) {
+				String name = info.getMenuPath().getLeaf().getName().trim();
+				String headline = "run(\"" + name +"\")";
+				String description = "<b>" + headline + "</b><p>" +
+						"<a href=\"https://imagej.net/Special:Search/" + name.replace(" ", "%20") + "\">Search imagej wiki for help</a>";
```

Comment:
How about harmonizing this with the behavior of the search bar's *Help* button? That one currently links to the command page directly: `https://imagej.net/<command-name>`

Reply:
I tried your suggestion first, but the very most lead to 404 errors, that's why I thought it's better like this...

Reply:
I am ambivalent. The goal should be to actually create all those missing wiki pages.